documentation/devon4ng-adding-custom-functionality.asciidoc

```diff
@@ -1189,6 +1200,7 @@ export class AccessCode {
     endTime?: string;
     visitorId: number;
     queueId: number;
+    content: any;
```

Comment:
Why not do we create a type in order not to use the `any` type? In general it is a bad practice.